src/org/exist/collections/triggers/XQueryTrigger.java

```diff
@@ -731,16 +707,16 @@ public void beforeCopyDocument(DBBroker broker, Txn txn, DocumentImpl document,
 		if (events.contains(TriggerEvents.EVENTS.COPY_DOCUMENT)) {
 			prepare(4, broker, txn, document.getURI(), newUri, false);
 		} else {
-		    execute(true, broker, txn, beforeCopyDocument, document.getURI());
+            execute(true, broker, txn, beforeCopyDocument, document.getURI(), newUri);
 	    }
 	}
 
 	@Override
-	public void afterCopyDocument(DBBroker broker, Txn txn, DocumentImpl document, XmldbURI newUri) throws TriggerException {
+    public void afterCopyDocument(DBBroker broker, Txn txn, DocumentImpl document, XmldbURI oldUri) throws TriggerException {
```

Comment:
if you implement base on interface then names is same, this implementation had typos